config/initializers/jsonapi_mimetypes.rb

```diff
@@ -0,0 +1,8 @@
+# Without this mimetype registration, controllers will not automatically parse JSON API params.
+module JSONAPI
+  MIMETYPE = "application/vnd.api+json"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.